iquip/apps/monitor.py

```diff
@@ -663,13 +907,24 @@ def __init__(
             self.dacControllerFrame.dacWidgets[name_].voltageSet.connect(
                 functools.partial(self._setDACVoltage, device, channel)
             )
+        for name_, info in ddsInfo.items():
+            device, channel = map(info.get, ("device", "channel"))
+            self.ddsControllerFrame.ddsWidgets[name_].profileSet.connect(
+                functools.partial(self._setDDSProfile, device, channel)
+            )
+            self.ddsControllerFrame.ddsWidgets[name_].attenuationSet.connect(
+                functools.partial(self._setDDSAttenuation, device, channel)
+            )
+            self.ddsControllerFrame.ddsWidgets[name_].switchClicked.connect(
+                functools.partial(self._setDDSSwitch, device, channel)
+            )
```

Comment:
How about defining a local variable, e.g., `widget = self.ddsControllerFrame.ddsWidgets[name_]`?
I'm not sure if each statement can be written in a single line though 😄

Reply:
Oh, it looks more legible! I will apply it.

Reply:
Still, we cannot write them in a single line, but it became legible😃